src/components/Pop/Popper.js

```diff
@@ -20,6 +21,7 @@ class Popper extends Component<Props> {
     modifiers: {},
     offset: 0,
     onClick: noop,
+    onContentClick: noop,
```

Comment:
@brettjonesdev Based on our convo! Adding the ability to close if you click the content. If you want!